code/script.js

```diff
@@ -218,16 +251,16 @@ const CHARACTERS = [
     img: 'images/julie.svg',
     hairColor: 'black',
     eyeColor: 'brown',
+    beard: false,
     glasses: true,
     hat: true,
     smoker: false,
   },
 ]
 
-// Global variables
-let secret, currentQuestion, charactersInPlay
+let secret, currentQuestion, charactersInPlay, playerName
+let attributeQuestions = 0
 
-// Draw the game board
 const generateBoard = () => {
```

Comment:
Maybe you can comment the functions as well since you did that with the eventlisteners, soundfiles ans the array?